### ISSUE TITLE
Unable to open the .dmg archive from developer.apple.com/fonts

### ISSUE THREAD

**Issue by Derpitron** [AppleDesignResources/SanFranciscoFont#8]
When I try to open the .dmg with both 7-zip and Winrar, it says that the archive is damaged or corrupted.

**Comment by windyboy1704** [comment]
You must use macOS to open that .dmg file

**Comment by windyboy1704** [comment]
https://github.com/windyboy1704/San-Francisco-family/tree/master/SF%20Pro
Last update as of June 2018.

**Comment by amatzen** [comment]
> I don't have a Mac though, what do I do?
> I can't set up a hackintosh at the time.

I don't think you are permitted to use the SF font outside the Apple eco system, since it only has been officially released for macOS. 

As [Wikipedia](https://en.wikipedia.org/wiki/San_Francisco_(sans-serif_typeface)) says: "Apple restricts the usage of the typeface by others. It is licensed to registered third-party developers only for the design and development of applications for Apple's platforms."

**Comment by yell0wsuit** [comment]
> > I don't have a Mac though, what do I do?
> > I can't set up a hackintosh at the time.
> 
> I don't think you are permitted to use the SF font outside the Apple eco system, since it only has been officially released for macOS.
> 
> As [Wikipedia](https://en.wikipedia.org/wiki/San_Francisco_(sans-serif_typeface)) says: "Apple restricts the usage of the typeface by others. It is licensed to registered third-party developers only for the design and development of applications for Apple's platforms."

Literally you cannot stop people from downloading the font. As long as you use it privately (without using it for commercial purposes), you're fine.

**Comment by amatzen** [comment]
> > > I don't have a Mac though, what do I do?
> > > I can't set up a hackintosh at the time.
> > 
> > 
> > I don't think you are permitted to use the SF font outside the Apple eco system, since it only has been officially released for macOS.
> > As [Wikipedia](https://en.wikipedia.org/wiki/San_Francisco_(sans-serif_typeface)) says: "Apple restricts the usage of the typeface by others. It is licensed to registered third-party developers only for the design and development of applications for Apple's platforms."
> 
> Literally you cannot stop people from downloading the font. As long as you use it privately (without using it for commercial purposes), you're fine.

It's a matter of respecting other's rights.

**Comment by nusserstudios** [comment]
Just download the zip file and install the otf files.
